styles/styles.css

```diff
@@ -324,44 +324,6 @@ main .section .default-content-wrapper h1 {
   font-size: 2em;
 }
 
-main .section.contact .default-content-wrapper h4 {
-  font-size: 1.5em;
-}
-
-main .section.contact .default-content-wrapper {
-  background-color: #FFF;
-  padding: 20px 10px 40px;
-}
-
-main .section.contact .default-content-wrapper p {
-  margin-bottom: 0;
-}
-
-main .section.contact .default-content-wrapper em {
-  font-size: 1em;
-  padding-bottom: 0;
-}
-
```

Comment:
These changes are to style 'contact' section on /digitech page which are still needed.